### PR TITLE
Issue #7157 fixed.

### PR DESCRIPTION
**Pull request by suraj-webkul**
Issue #7157 fixed.
Add drag and drop option

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/images.blade.php [hunk 1]
```diff
@@ -159,7 +165,9 @@ class="preview"
                 return {
                     imageData: '',
 
-                    new_image: 0
+                    newImage: 0,
+
```
Comment (akathuria289) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/images.blade.php:169]: Remove this space @suraj-webkul

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
Add the same for the category section too.

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Not working** in for categories.

**Comment by suraj-webkul** [comment]
Please publish assets then try again

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
**Not working,** if trying to **drop** the **variants** image.

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/images.blade.php (hunk 1 above)
Comment (suraj-webkul) [reply packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/images.blade.php]: space removed

**Review by akathuria289 — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
This is **working fine.**

**Review by Sanjaybhattwebkul — DISMISSED** [review]

**Review by akathuria289 — DISMISSED** [review]

File: packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/variations.blade.php [hunk 2]
```diff
@@ -341,14 +345,14 @@ class="control"
...
+        let superAttributes = @json(app('\Webkul\Product\Repositories\ProductRepository')->getSuperAttributes($product));
         let variants = @json($product->variants);
 
         Vue.component('variant-form', {
             data: function () {
                 return {
                     variant: {},
-                    super_attributes: super_attributes
+                    superAttributes: superAttributes
                 }
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/variations.blade.php:356]: Optimize this. Pass it through AJAX request.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Comment by Amitk-webkul** [comment]
@suraj-webkul,

Resolve the conflicts.

**Comment by suraj-webkul** [comment]
[Amitk-webkul](https://github.com/Amitk-webkul) Conflicts Resolved

**Comment by Sanjaybhattwebkul** [comment]
@suraj-webkul 
any update?

**Pull request closed without merging**